docs/tutorials/kafka-streams-examples.rst

```diff
@@ -0,0 +1,247 @@
+    # the output is pretty-printed so that it's easier to read:
+    [
+      {
+        "host": "localhost",
+        "port": 7070,
+        "storeNames": [
+          "all-songs",
+          "song-play-count",
+          "top-five-songs",
+          "top-five-songs-by-genre"
+        ]
+      }
+    ]
+
+**REST API example 2: get the latest Top 5 songs across all music genres:**
+
+.. sourcecode:: bash
+
+    # Mac and Windows users
+    $ curl -sXGET http://`docker-machine ip confluent`:7070/kafka-music/charts/top-five
+
+    # Linux users
+    $ curl -sXGET http://localhost:7070/kafka-music/charts/top-five
+
+    # You should see output similar to following, though here
+    # the output is pretty-printed so that it's easier to read:
+    [
+      {
+        "artist": "Jello Biafra And The Guantanamo School Of Medicine",
+        "album": "The Audacity Of Hype",
+        "name": "Three Strikes",
+        "plays": 70
+      },
+      {
+        "artist": "Hilltop Hoods",
+        "album": "The Calling",
+        "name": "The Calling",
+        "plays": 67
+      },
+
+      ... rest omitted...
+    ]
+
+The REST API exposed by the
+:cp-examples:`Kafka Music application|src/main/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExample.java`
+supports further operations.  See the
+:cp-examples:`top-level instructions in its source code|src/main/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExample.java`
+for details.
+
+Once you're done playing around you can stop all the services and containers with:
+
+.. sourcecode:: bash
+
+    $ docker-compose down
+
+We hope you enjoyed this tutorial!
+
+
+Running further Confluent demo applications for the Kafka Streams API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The container named ``kafka-music-application``, which runs the Kafka Music demo application, actually contains all of
+Confluent's `Kafka Streams demo applications <https://github.com/confluentinc/examples>`__.  The demo applications are
+packaged in the fat jar at ``/app/streams-examples-3.2.0-standalone.jar`` inside this container.  This means you can
+easily run any of these applications from inside the container via a command similar to:
+
+.. sourcecode:: bash
+
+    # Example: Launch the WordCount demo application (inside the `kafka-music-application` container)
+    $ docker-compose exec kafka-music-application \
+            java -cp /app/streams-examples-3.2.0-standalone.jar \
+            io.confluent.examples.streams.WordCountLambdaExample \
+            localhost:29092
+
+Of course you can also modify the tutorial's ``docker-compose.yml`` for repeatable deployments.
+
+Note that you must follow the full instructions of each demo application (see its respective source code at
+https://github.com/confluentinc/examples).  These instructions include, for example, the creation of the application's
```

Comment:
No, the link does not need markup -- works as is (I did check because I wondered myself :) ).